### PR TITLE
Send and receive data from an API

### PR DESCRIPTION
**Pull request by Bornittah**
**Implementation**

- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,31 @@
...
+});
+
+refresh.addEventListener('click', async () => {
+  await showScoresToUI(scores);
+});
+
+window.addEventListener('DOMContentLoaded', async (event) => {
+  event.preventDefault();
+  await showScoresToUI(scores);
+});
```
Comment (vechicin) [inline src/index.js:31]: I could only submit a score once. Every time after that, it seemed like the score I submitted was posted to the API but it is not being fetched correctly. Please fix this.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Bornittah ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Proper `async` and `await` use
- [x] Proper arrow function use 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Bornittah) [reply src/index.js:31]: Kindly ellaborate more, may be with a short video. i didnt get well where there error is. Thanks

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: 

Hi @Bornittah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**